client/src/components/RelatedAndComparison/assets/RelatedProducts.jsx

```diff
@@ -1,65 +1,53 @@
 /* eslint-disable no-plusplus */
 /* eslint-disable prefer-destructuring */
 /* eslint-disable react/prop-types */
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import { FaArrowAltCircleLeft, FaArrowAltCircleRight } from 'react-icons/fa';
+import Images from './Images.jsx';
 
 const RelatedProducts = ({ styles, handleClick }) => {
   const [currentCard, setCurrentCard] = useState(0);
-  const [cardDisplay, setCardDisplay] = useState([]);
   const length = styles.length;
 
-  useEffect(() => {
-    const display = [];
-
-    if (styles.length >= 1) {
-      for (let i = currentCard; i < (currentCard + 5); i++) {
-        if (currentCard === 0) {
-          display.push(styles[i]);
-        } else {
-          display.push(styles[i - 1]);
-        }
-      }
-    }
-    setCardDisplay(display);
-  }, [styles, currentCard]);
-
-  const prevSlide = () => {
+  const prevCard = () => {
```

Comment:
For the future, you might want to work on the right arrow not showing if there are no new cards to render. That way when you click the right arrow multiple, cards do not disappear towards the end.